### DISCUSSION TITLE
Embrace GitOps in the installation guide

### DISCUSSION THREAD

**Discussion by vaskozl** [fluxcd/flux2#312] [Proposals]
First off I have to say this project is amazing and I am already using it for my home.

However the conclusion of the Installation guide (https://toolkit.fluxcd.io/guides/installation/), stuck out to me:
```
kustomize build github.com/fluxcd/toolkit/manifests/monitoring | kubectl apply -f-
```
Is there any reason we don't add a source and apply it with a kustomization? The instructions immediately made me wonder as to whether doing so is actually possible and if there is some limitation which warrants piping into `kubectl` as above.

I feel many people turn to flux exactly because piping into kubectl feels uncomfortable.

(There are a few other places in the documentation where `gotk create` is used without `--export`, which confused me, given that `gotk bootstrap` does commit its changes to the repo and the instruction seem to set out to create a pure GitOps setup.)

**Comment by stefanprodan** [comment]
@Vaskozl you are right, the monitoring addons can be installed via a Kustomization and we should make it very clear that when using `create` without export is just for dev/test mode.

**Comment by stealthybox** [comment]
Here's an example of committing the "child" `GitRepo` and `Kustomization` for the monitoring stack to the gotk-system fleet repository from yesterday's community demo:
https://youtu.be/0v5bjysXTL8?t=2213
